oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/Commit.java

```diff
@@ -352,7 +352,10 @@ private void applyToDocumentStore(RevisionVector baseBranchRevision)
             JournalEntry doc = JOURNAL.newDocument(store);
             doc.modified(modifiedNodes);
             Revision r = revision.asBranchRevision();
-            store.create(JOURNAL, singletonList(doc.asUpdateOp(r)));
+            boolean success = store.create(JOURNAL, singletonList(doc.asUpdateOp(r)));
+            if (!success) {
+                LOG.error("Failed to update journal for revision {}", r);
```

Comment:
Was wondering if we should also log the `doc` - it would contain the modified paths (perhaps useful in analysing and presumably not leaking sensitive information)?

Reply:
The whole doc, as with "toString()"?

Reply:
It would have been the whole doc yes.

Alternatively, we could also do two logs : the existing LOG.error and an additional LOG.debug with the doc... (such that it wouldn't log by default, but it could be enabled)... (but I guess it's either way more of a detail)